File: INICIANTE/JAVASCRIPT/1064.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift(); };

var contador = 0
var soma = 0

for(i=0;i<6;i++){
    var x = parseFloat(prompt(""))
    if(x > 0){
    contador = contador + 1
    soma = soma + x
    }
}
media = soma/contador
console.log(contador + " valores positivos")
console.log(media.toFixed(1))

File: INICIANTE/JAVA/1064.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int contador =	0;
        double soma = 0;

        for(int i=0;i<6;i++){
            double x = sc.nextDouble();
            if(x > 0){
            contador ++;
            soma += x;
            }
        }
        double media = soma/contador;

        System.out.println(contador + " valores positivos");
        System.out.println(String.format("%.1f", media));
        sc.close();
    }
}
